src/components/shell/shell.js

```diff
@@ -325,11 +326,13 @@ class Shell extends Component {
    */
   _startHoverTimeout(): void {
     this._clearHoverTimeout();
-    this.hoverTimeout = setTimeout(() => {
-      if (this._canEndHoverState()) {
-        this._updatePlayerHover(false);
-      }
-    }, this.props.hoverTimeout || CONTROL_BAR_HOVER_DEFAULT_TIMEOUT);
```

Comment:
so need to delete `CONTROL_BAR_HOVER_DEFAULT_TIMEOUT`

Reply:
better to check why it exported at all

Reply:
Maybe a customer who adding custom UI can take the shell and import this data from the shell.
It'll break who use this constant

Reply:
It was added here - https://kaltura.atlassian.net/browse/FEC-7373
@yairans  I don't think we're able to know if it was on purpose or not...(it looks like it's not)

Reply:
you can remove it as no one can import it outside the player
see https://github.com/kaltura/playkit-js-ui/blob/master/src/components/shell/index.js
we exporting `Shell` only (for all we have - `export *`)